tutorials/ph-mult-rasters/index.md

```diff
@@ -0,0 +1,131 @@
+---
+title: Using Earth Engine to export multiple rasters based on user-defined dates
+description: In this tutorial, we will learn how to export multiple land surface temperature rasters for the same month over multiple years.
+author: hiyer09
+tags: exporting data, rasters, loops, Africa, public health
+date_published: 2019-09-18
+---
+
+[Open In Code Editor](https://code.earthengine.google.com/2d3ad0c07630e858f0274d86a207d994)
+
+It is desirable to pull multiple raster images for environmental public health research. Typically the research question will guide the temporal resolution, but in most cases having multiple images over the course of the year is useful to interrogate possible seasonal patterns in exposure-disease relationships.
```

Comment:
Suggestion for clarity: change 'pull' to 'export'

Reply:
Done